backup-loop.sh

```diff
@@ -28,42 +60,35 @@ fi
 log INFO "waiting for rcon readiness..."
 while true; do
   rcon-cli save-on >& /dev/null && break
-
   sleep 10
 done
-log INFO "waiting initial delay of ${INITIAL_DELAY} seconds..."
-sleep ${INITIAL_DELAY}
 
 while true; do
   ts=$(date -u +"%Y%m%d-%H%M%S")
 
-  rcon-cli save-off
-  if [ $? = 0 ]; then
-
-    rcon-cli save-all
-    if [ $? = 0 ]; then
+  if rcon-cli save-off; then
+    if rcon-cli save-all; then
 
       outFile="${DEST_DIR}/${BACKUP_NAME}-${ts}.tgz"
       log INFO "backing up content in $(pwd) to ${outFile}"
-      tar cz -f ${outFile} ${backupSet} ${excludes}
-      if [ $? != 0 ]; then
+      # shellcheck disable=SC2086
+      if tar cz -f "${outFile}" ${backupSet} ${excludes}; then
         log ERROR "backup failed"
       else
         log INFO "successfully backed up"
       fi
 
     fi
-
     rcon-cli save-on
```

Comment:
Should we handle error here? If so, how? Retries or just error message and continue with the script?

Reply:
On the `rcon-cli save-on`? Probably should but good question on how to handle that. Retrying does seem like a good choice here.